assets/airports/airportLoadList.js

```diff
@@ -71,6 +71,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             name: 'Berlin Tegel Airport',
             wip: true
         },
+        {
+            icao: 'efhk',
+            level: 'medium',
```

Comment:
`level: 'easy',`

Reply:
IIRC, I initially had it easy until somebody suggested medium in a review :-). For me, either is fine.